spreadsheets/collision_rank.yaml

```diff
@@ -0,0 +1,943 @@
+- {layer: 'roads', kind: 'aeroway'}
+# for shields & etc
+- _reserved: {from: 1551, to: 1561}
+- {layer: 'pois', kind: 'courthouse'}
+- {layer: 'pois', kind: 'lighthouse'}
+- {layer: 'pois', kind: 'obelisk'}
+# reserve for kind_details
+- _reserved: {from: 1565, to: 1575}
+- {layer: 'pois', kind: 'police'}
+- {layer: 'pois', kind: 'post_office'}
+- {layer: 'pois', kind: 'ruins'}
+# reserve for kind_details
+- _reserved: {from: 1579, to: 1589}
+- {layer: 'pois', kind: 'watermill'}
+- {layer: 'pois', kind: 'windmill'}
+- {layer: 'places', kind: 'neighbourhood'}
+- {layer: 'pois', kind: 'border_control'}
+- {layer: 'pois', kind: 'customs'}
+- {layer: 'earth', kind: 'ridge'}
+- {layer: 'pois', kind: 'boatyard'}
+- {layer: 'pois', kind: 'shipyard'}
+- {layer: 'pois', kind: 'spring'}
+- {layer: 'pois', kind: 'substation'}
+- {layer: 'pois', kind: 'gate'}
+# reserve for kind_details
+- _reserved: {from: 1601, to: 1611}
+- {layer: 'pois', kind: 'horse_riding'}
+- {layer: 'pois', kind: 'wharf'}
+- {layer: 'pois', kind: 'adult_gaming_centre'}
+- {layer: 'pois', kind: 'egress'}
+- {layer: 'pois', kind: 'parking'}
+- {layer: 'pois', kind: 'parking_garage'}
+- {layer: 'pois', kind: 'put_in'}
+- {layer: 'pois', kind: 'put_in_egress'}
+- {layer: 'pois', kind: 'quay'}
+- {layer: 'pois', kind: 'saddle'}
+- {layer: 'pois', kind: 'arts_centre'}
+- {layer: 'pois', kind: 'bar'}
+# reserve for kind_details
+- _reserved: {from: 1624, to: 1724}
+- {layer: 'pois', kind: 'bicycle'}
+- {layer: 'pois', kind: 'biergarten'}
+# reserve for kind_details
+- _reserved: {from: 1727, to: 1827}
+- {layer: 'pois', kind: 'cafe'}
+# reserve for kind_details
+- _reserved: {from: 1829, to: 1929}
+- {layer: 'pois', kind: 'fast_food'}
+# reserve for kind_details
+- _reserved: {from: 1931, to: 2031}
+- {layer: 'pois', kind: 'field_hospital'}
+# reserve for kind_details
+- _reserved: {from: 2033, to: 2053}
+- {layer: 'pois', kind: 'gallery'}
+- {layer: 'pois', kind: 'health_centre'}
+# reserve for kind_details
+- _reserved: {from: 2056, to: 2076}
+- {layer: 'pois', kind: 'nightclub'}
+# reserve for kind_details
+- _reserved: {from: 2078, to: 2178}
+- {layer: 'pois', kind: 'observatory'}
+- {layer: 'pois', kind: 'outdoor'}
+- {layer: 'pois', kind: 'pub'}
+# reserve for kind_details
+- _reserved: {from: 2182, to: 2282}
+- {layer: 'pois', kind: 'restaurant'}
+# reserve for kind_details
+- _reserved: {from: 2284, to: 2384}
+- {layer: 'pois', kind: 'theatre'}
+- {layer: 'pois', kind: 'toll_booth'}
+- {layer: 'pois', kind: 'trailhead'}
+- {layer: 'pois', kind: 'viewpoint'}
+- {layer: 'pois', kind: 'water_tower'}
+- {layer: 'pois', kind: 'wilderness_hut'}
+- {layer: 'pois', kind: 'nursing_home'}
+# reserve for kind_details
+- _reserved: {from: 2392, to: 2412}
+- {layer: 'pois', kind: 'telescope'}
+- {layer: 'landuse', kind: 'city_wall'}
+- {layer: 'water', kind: 'riverbank'}
+- {layer: 'earth', kind: 'cliff'}
+- {layer: 'earth', kind: 'arete'}
+- {layer: 'boundaries', kind: 'maritime'}
```

Comment:
This was in the CSV list output, but does it get transformed into `maritime_boundary` = True somewhere else in the code?

Reply:
Yes! Well spotted, thanks. Fixed in 998d0b3 and tests added.